mantle/kola/harness.go

```diff
@@ -277,12 +277,13 @@ func testRequiresInternet(test *register.Test) bool {
 }
 
 type DenyListObj struct {
-	Pattern    string   `yaml:"pattern"`
-	Tracker    string   `yaml:"tracker"`
-	Streams    []string `yaml:"streams"`
-	Arches     []string `yaml:"arches"`
-	Platforms  []string `yaml:"platforms"`
-	SnoozeDate string   `yaml:"snooze"`
+	Pattern     string   `yaml:"pattern"`
+	Tracker     string   `yaml:"tracker"`
+	TrackerList []string `yaml:"tracker_list"`
```

Comment:
```suggestion
	Tracker     []string `yaml:"tracker"`
```
The new functionality makes sense to me. I think we can change `Tracker` to `[]string` rather than adding a `TrackerList` which seems to be closer to the assumed functionality in the linked PRs. If we prefer to have a separate field, that works too!

Reply:
I tried the wholesale change from `string` to `[]string`, but the YAML module is strict about the type (well, all of Golang is strictly typed...) so it blows up trying to unmarhsal the data:

`cannot unmarshal !!seq into main.DenyListObj`

See this example snippet I was playing with - https://play.golang.org/p/WxafbrQGuNe

Reply:
Yep you're right that functionality isn't present, ignore the above suggestion.